### PR TITLE
Splitter : add test for resizing multiple panes

### PR DESCRIPTION
**Pull request by jdvictoria**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.width.tests.js [hunk 1]
```diff
@@ -66,10 +66,10 @@ QUnit.module('Tabs width', () => {
 
             if(options.width > 250) {
                 this.assert.strictEqual(getOuterWidth(this._getTabItem(0)) <= 140, true, getOuterWidth(this._getTabItem(0)) + ' <= 140');
-                this.assert.strictEqual(getOuterWidth(this._getTabItem(1)) <= 160, true, getOuterWidth(this._getTabItem(1)) + ' <= 160');
```
Comment (AlexanderMoiseev) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.width.tests.js]: why did you change tabPanel tests?

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js [hunk 2]
```diff
@@ -168,6 +168,61 @@ QUnit.module('Resizing', moduleConfig, () => {
...
+
+            const items = this.$element.find(`.${SPLITTER_ITEM_CLASS}`);
+
+            const pointer = pointerMock(this.getResizeHandles().eq(0));
+            pointer.start().dragStart().drag(50, 50).dragEnd();
+
+            assertLayout(items, ['38.4146', '28.252', '33.3333'], assert);
+        });
+
+        QUnit.test('resize handles visibility with multiple items in dataSource', function(assert) {
```
Comment (AlexanderMoiseev) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js]: i believe we have these kind of tests already

**Comment by AlexanderMoiseev** [comment]
change the title please according to our style guide

**Review by AlexanderMoiseev — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]
